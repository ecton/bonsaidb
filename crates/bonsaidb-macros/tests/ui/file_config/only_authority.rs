use bonsaidb::files::FileConfig;

#[derive(FileConfig)]
#[file_config(authority = "authority")]
struct Test;

fn main() {}
