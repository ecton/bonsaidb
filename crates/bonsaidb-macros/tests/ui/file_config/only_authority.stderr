error: if `authority` is specified, `files_name` and `blocks_name need to be provided as well
 --> tests/ui/file_config/only_authority.rs:3:10
  |
3 | #[derive(FileConfig)]
  |          ^^^^^^^^^^
  |
  = note: this error originates in the derive macro `FileConfig` (in Nightly builds, run with -Z macro-backtrace for more info)
